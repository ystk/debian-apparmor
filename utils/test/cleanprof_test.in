# A simple test comment which will persist
#include <tunables/global>

/usr/bin/a/simple/cleanprof/test/profile {
	# Just for the heck of it, this comment wont see the day of light
	#include <abstractions/base>
	#Below rule comes from abstractions/base
	allow /usr/share/X11/locale/**  r,
	allow /home/*/** r,
	allow /home/foo/bar r,
	allow /home/foo/** w,
}

/usr/bin/other/cleanprof/test/profile {
	# This one shouldn't be affected by the processing
	# However this comment will be wiped, need to change that
	allow /home/*/** rw,
	allow /home/foo/bar r,
}