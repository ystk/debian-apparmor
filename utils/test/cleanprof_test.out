#include <tunables/global>

# A simple test comment which will persist


/usr/bin/a/simple/cleanprof/test/profile {
  #include <abstractions/base>

  /home/*/** r,
  /home/foo/** w,

}
/usr/bin/other/cleanprof/test/profile {
  /home/*/** rw,
  /home/foo/bar r,

}
